File: Java/2_Selection Sort/src/SelectionSort.java
public class SelectionSort {

    public static void selectionSort(int[] arr){
        for(int k = 0; k < arr.length - 1; k++){
            int min = k;
            for(int j = k + 1; j < arr.length; j++){
                if(arr[j] < arr[min]){
                    min = j;
                }
            }
            int temp = arr[min];
            arr[min] = arr[k];
            arr[k] = temp;
        }
    }

    public static void printArray(int[] arr){
        for(int i = 0; i < arr.length; i++){
            System.out.print(arr[i] + " ");
        }
    }

    public static void main(String[] args) {
        int[] arr = {64, 25, 12, 22, 11};
        selectionSort(arr);
        printArray(arr);
    }
}

File: Javascript/2_Selection Sort/Selection Sort.js
function selectionSort(array) {
    for(var k = 0; k < array.length; k++){
        var min = k;
        for(var j = k + 1; j < array.length; j++){
            if(array[j] < array[min]){
                min = j;
            }
        }
        var temp = array[k];
        array[k] = array[min];
        array[min] = temp;
    }
}

var array = [64, 25, 12, 22, 11];
selectionSort(array);
console.log(array);